erpnext/controllers/selling_controller.py

```diff
@@ -857,3 +857,32 @@ def update_customer_name_from_master(doctype, name):
 
 	doc.notify_update()
 	doc.save_version()
+
+
+@frappe.whitelist()
+def update_vehicle_details_from_master(doctype, name):
+	from automotive.controllers.vehicle_transaction_controller import get_vehicle_details
```

Comment:
Do not import automotive app in erpnext, use `set_missing_applies_to_details` instead of `get_vehicle_details`